src/installer/pkg/sfx/Directory.Build.props

```diff
@@ -10,6 +10,8 @@
     <ProductBrandPrefix>Microsoft .NET</ProductBrandPrefix>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <UseRuntimePackageDisclaimer>true</UseRuntimePackageDisclaimer>
+    <!-- We don't have any sources here, so don't run analyzers. -->
+    <RunAnalyzers>false</RunAnalyzers>
```

Comment:
You could also condition on shared framework packages with the following property: https://github.com/dotnet/runtime/blob/64243bbf5e9ee53c0c4c5678f2cd8c7f1c9b4f6f/Directory.Build.props#L408

Reply:
I can't as that property isn't set early enough to be available in Directory.Build.props files. I tried it and it didn't work.

Reply:
Consider setting it in analyzers.targets. At that point, the property should already be set.